app/schemas/storage.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+from uuid import UUID
+
+
+class StorageCreate(BaseModel):
+    id: UUID
+    title: str
```

Comment:
`id`는 자동 생성이라 없어도 되지 않을까요? 그리고 category_id는 필요 없는지 궁금합니당

```suggestion
class StorageCreate(BaseModel):
    title: str
```

Reply:
@GideokKim 
아 id 는 필요없습니다! category_id 와 user_id는 파라미터로 받아서 추가해주었습니다!